case-studies/docker/docker-case-study.md

```diff
@@ -0,0 +1,265 @@
+This makes much faster to boot and it do not need [kernel](https://en.wikipedia.org/wiki/KERNAL) to run, Docker will then use less H/W resources to run. Docker take the application, Dependencies to run a Container and the docker engine run with [REST API](https://en.wikipedia.org/wiki/Representational_state_transfer) and [CLI](https://en.wikipedia.org/wiki/Command-line_interface) to let user connect to the container. For developer it's good development platform for building good workflow, Version Control on Software, like [NodeJS](https://en.wikipedia.org/wiki/Node.js), [Java](https://en.wikipedia.org/wiki/Java_(programming_language)), [python](https://en.wikipedia.org/wiki/Python_(programming_language)), [MariaDB](https://en.wikipedia.org/wiki/MariaDB), [SQL](https://en.wikipedia.org/wiki/SQL). but not only developer use this platform, Cyber-security resource, Cloud base Platform. Docker is good tool in the IT world today. Most Operative system, Client base and server has support for docker. today Docker has one of the biggest Container image libraries in the world on [Docker Hub](https://hub.docker.com/). 
+
+
+
+## Brief History 
+
+when docker launched in 2013 it’s, was a Linux-based kernel containerization technology. Only Linux user was using docker before docker partner up with [Redhat](https://www.redhat.com/en) and This may be the most prosperous partnership Docker has made because this led to even bigger partnership in docker history, [Microsoft](https://www.microsoft.com/) wants to make this technology ready for Windows users and Windows server. Before Microsoft made partnership with Docker [LinkedIn](https://www.linkedin.com/) so that Docker had grew by 160% present by 2016. By 2019 docker and Microsoft released docker to windows and windows server.  
+
+
+
+- 2013 Docker and Red Hat Announced a collaboration around Fedora, Red Hat Enterprise, OpenShift
+- 2014 October Microsoft announced the intergration of the Docker engine into Windows Server, Support for Docker client in windows. 
+- November 2014: Docker container services ware announced for the Amazon elastic Computer cloud. 
+- 2014 November: Docker annonced a partership with Stratoscal.
+- 2014 December IBM anounced a partnershipt with Docker that enables Docker to integrate more to IBM cloud.
+- 2015 December: Oracle Cloud added Docker container support after acquiring SlackEngine.  
+- 2016 April: Windoc and independent software vendor released a port of open source project to windows, supporting windows 2012 R2 
+- 2016 May: Analysis of Linkedin profile mentoions showed Docker grew by 160% in 2016
+- 2019 May Announced the second verison of windows subsystem for link (WSL) from microsoft
+- 2019 aug Annoced a backport of WLS2 to windows from microsoft 
```

Comment:
Please check spelling, I recommend using code spell checker extension for VS code